### PR TITLE
11.0 mig crm claim type

### PR DESCRIPTION
**Pull request by larmar**
I have upgraded this version 11

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Thanks, please squash together "OCA Transbot..." commits that can be joined.

**Comment by larmar** [comment]
Should 
[FIX] remove en.po that was erroneously created by transbot [Stéphane Bidoul (ACSONE)] also be squashed in Transbot
Or leave one Transibot before and one after ?

**Comment by pedrobaeza** [comment]
Yes, please squash it also.

**Review by keshrath — CHANGES_REQUESTED** [review]
Exception: Module loading crm_claim_type failed: file crm_claim_type\security/ir.model.access.csv could not be processed:
No matching record found for external id 'base.group_sale_manager' in field 'Group'
No matching record found for external id 'base.group_sale_salesman' in field 'Group'

--> sales_team.group_sale_manager
--> sales_team.group_sale_salesman

odoo.tools.convert.ParseError: "FEHLER:  NULL-Wert in Spalte »name« verletzt Not-Null-Constraint
DETAIL:  Fehlgeschlagene Zeile enthält (317, null, ir.actions.act_window, null, null, action, 1, 2018-04-12 15:34:05.916973, 1, 2018-04-12 15:34:05.916973, null, null, 
                {"search_default_user_id":uid, "stage_type":'cl..., null, null, null, current, tree,form, form, null, 80, null, f, t, f).
" while parsing file:/c:/users/mathias/workspace/crm_claim_type/views/crm_claim_stage.xml:4, near
```
<record id="crm_claim.crm_case_categ_claim0" model="ir.actions.act_window">
            <field name="context">
                {"search_default_user_id":uid, "stage_type":'claim'}
            </field>
        </record>
```

**Comment by ecino** [comment]
What's the status of this?

File: crm_claim_type/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# © 2015 Vauxoo: Yanina Aular <yani@vauxoo.com>, Osval Reyes <osval@vauxoo.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    'name': 'CRM Claim Types',
+    'category': 'Customer Relationship Management',
+    'summary': 'Claim types for CRM',
+    'author': 'Odoo Community Association (OCA),'
+              'Vauxoo,'
+              'Ursa Information Systems',
```
Comment (MeetKD) [inline crm_claim_type/__manifest__.py:10]: I guess OCA as author should be at last.

**Review by MeetKD — COMMENTED** [review]
If you are going to work on it please attend this minor comment.

**Comment by quentingigon** [comment]
Will this module be merged soon ?

**Comment by ecino** [comment]
@eicher31 Could you test and review this one as well?

**Pull request closed without merging**